scripts/zkai_native_d128_compressed_outer_statement_binary_accounting_gate.py

```
@@ -0,0 +1,659 @@
+        "proof_payload_kind",
+        "size_constants",
+        "profile_row",
+        "aggregate",
+        "validation_commands",
+        "payload_commitment",
+    }
+    mutation_keys = {"mutation_cases", "mutations_checked", "mutations_rejected", "all_mutations_rejected"}
+    if allow_missing_mutation_summary:
+        allowed_keys = expected_keys | mutation_keys
+        if not set(payload).issubset(allowed_keys) or not expected_keys.issubset(payload):
+            raise BinaryTypedProofAccountingGateError("payload field drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)